package/entrypoint.sh

```diff
@@ -6,4 +6,7 @@ if [ ! -e /run/secrets/kubernetes.io/serviceaccount ] && [ ! -e /dev/kmsg ]; the
     exit 1
 fi
 rm -f /var/lib/rancher/k3s/server/cred/node-passwd
+if [ -x "$(command -v c_rehash)" ]; then
+  c_rehash
+fi
```

Comment:
I agree with the check here for local development, but shouldn't we fail if the executable is not found or in PATH?

Reply:
If a user doesn't set `additionalTrustedCAs` to `true`, then we don't want to fail because `c_rehash` isn't on the `PATH`.